gui/Cargo.toml

```diff
@@ -41,3 +41,6 @@ uds_windows = "0.1.5"
 
 [dev-dependencies]
 tokio = {version = "1.9.0", features = ["rt", "macros"]}
+
+[patch.crates-io]
+miniscript = { git = "https://github.com/darosior/rust-miniscript", branch = "multipath_descriptors_on_9.0" }
```

Comment:
But you don't have a `miniscript` dependency? Did you intend to do that for Liana?

Reply:
If I do not add this patch liana gui fetch miniscript 9.0.0 and the  build fail.

Reply:
```
liana/gui on  master is 📦 v0.0.1 via 🦀 v1.65.0 
❯ cargo update -p liana
    Updating git repository `https://github.com/revault/liana`
    Updating crates.io index
    Updating liana v0.0.1 (https://github.com/revault/liana?branch=master#38e342c8) -> #9ba5f673
      Adding miniscript v9.0.0
    Removing miniscript v8.0.0 (https://github.com/darosior/rust-miniscript?branch=multipath_descriptors_on_8.0#a63d5a26)

liana/gui on  master [!] is 📦 v0.0.1 via 🦀 v1.65.0 
❯ cargo build
   Compiling liana v0.0.1 (https://github.com/revault/liana?branch=master#9ba5f673)
error[E0407]: method `num_der_paths` is not a member of trait `MiniscriptKey`
   --> /home/edouard/.cargo/git/checkouts/liana-f072ff0678f8d7b0/9ba5f67/src/descriptors.rs:143:5
    |
143 | /     fn num_der_paths(&self) -> usize {
144 | |         0
145 | |     }
    | |_____^ not a member of trait `MiniscriptKey`

error[E0599]: no variant or associated item named `MultiXPub` found for enum `DescriptorPublicKey` in the current scope
   --> /home/edouard/.cargo/git/checkouts/liana-f072ff0678f8d7b0/9ba5f67/src/descriptors.rs:192:42
    |
192 |         descriptor::DescriptorPublicKey::MultiXPub(ref xpub) => {
    |                                          ^^^^^^^^^ variant or associated item not found in `DescriptorPublicKey`

error[E0599]: no method named `into_single_descriptors` found for enum `Descriptor` in the current scope
   --> /home/edouard/.cargo/git/checkouts/liana-f072ff0678f8d7b0/9ba5f67/src/descriptors.rs:302:14
    |
302 |             .into_single_descriptors()
    |              ^^^^^^^^^^^^^^^^^^^^^^^ method not found in `Descriptor<DescriptorPublicKey>`

error[E0599]: no method named `into_single_descriptors` found for enum `Descriptor` in the current scope
   --> /home/edouard/.cargo/git/checkouts/liana-f072ff0678f8d7b0/9ba5f67/src/descriptors.rs:386:14
    |
386 |             .into_single_descriptors()
    |              ^^^^^^^^^^^^^^^^^^^^^^^ method not found in `Descriptor<DescriptorPublicKey>`

error[E0599]: no variant or associated item named `MultiXPub` found for enum `DescriptorPublicKey` in the current scope
   --> /home/edouard/.cargo/git/checkouts/liana-f072ff0678f8d7b0/9ba5f67/src/descriptors.rs:403:53
    |
403 |             if let descriptor::DescriptorPublicKey::MultiXPub(xpub) = xpub {
    |                                                     ^^^^^^^^^ variant or associated item not found in `DescriptorPublicKey`

error[E0599]: no method named `expect` found for struct `DefiniteDescriptorKey` in the current scope
   --> /home/edouard/.cargo/git/checkouts/liana-f072ff0678f8d7b0/9ba5f67/src/descriptors.rs:492:22
    |
492 |                     .expect("We disallow multipath keys.");
    |                      ^^^^^^ method not found in `DefiniteDescriptorKey`

Some errors have detailed explanations: E0407, E0599.
For more information about an error, try `rustc --explain E0407`.
error: could not compile `liana` due to 6 previous errors

```

Reply:
Then this is an issue on the Liana side! Obviously we are not going to require all our users to set the patch in their Cargo.toml..... I'm on it..

Reply:
See https://github.com/revault/liana/pull/178.